2024-03-27/agenda.md

```diff
@@ -1,11 +1,19 @@
 # Agenda for MDN Community meeting 27th March '24
 
+**Join details:** See 'where' in [the calendar invite](https://calendar.google.com/calendar/u/0/embed?src=c_4656dd7c36825e2be115c0e7992191d550d16edcec37151eb6018581f654727b@group.calendar.google.com) for the call URL.
+
 ## Intro & overview of agenda
 
+- Rules: use the chat for questions or to request to speak so that we're talking in-turns.
```

Comment:
something like this?
```suggestion
- Rules: Please use the chat for questions or if you wish to speak, so that we can ensure everyone has a chance to talk.
```